src/test/scala/com/cognite/sdk/scala/common/SdkTestSpec.scala

```diff
@@ -41,18 +46,36 @@ class LoggingSttpBackend[F[_], +P](delegate: SttpBackend[F, P]) extends SttpBack
   override def responseMonad: MonadError[F] = delegate.responseMonad
 }
 
-abstract class SdkTestSpec extends AnyFlatSpec with Matchers {
+abstract class SdkTestSpec extends AnyFlatSpec with Matchers with OptionValues {
+  implicit val authSttpBackend: SttpBackend[IO, Any] = AsyncHttpClientCatsBackend[IO]().unsafeRunSync()
   // Use this if you need request logs for debugging: new LoggingSttpBackend[Id, Nothing](sttpBackend)
-  lazy val client: GenericClient[Id] = GenericClient.forAuth[Id](
-    "scala-sdk-test", auth)(implicitly, sttpBackend)
-
-  lazy val projectName: String = client.login.status().project
+  lazy val client: GenericClient[Id] = GenericClient[Id](
+    "scala-sdk-test",
+    projectName,
+    baseUrl,
+    auth
+  )
 
   def shortRandom(): String = UUID.randomUUID().toString.substring(0, 8)
+  lazy val projectName: String = sys.env("TEST_PROJECT_NAME")
+  lazy val baseUrl: String = sys.env("TEST_BASE_URL")
+  private lazy val tenant: String = sys.env("TEST_AAD_TENANT")
+  private lazy val clientId: String = sys.env("TEST_CLIENT_ID")
+  private lazy val clientSecret: String = sys.env("TEST_CLIENT_SECRET")
+  private lazy val scopes: List[String] = sys.env("TEST_SCOPES").split(";").toList
```

Comment:
I think some of these can be hard coded here, such as projectName, scopes if there is no specific reason. Not to require extra environment variables 🤔